test/test_error_handlers.py

```
@@ -0,0 +1,74 @@
+    def test_listeners_handler(self):
+        channel = FakeChannel()
+        channel.register_listener_error_handler(self.listeners_handler_mock)
+        my_listener = BrokenListener()
+        my_listener.start()
+        channel.register_listener(my_listener)
+        msg = Message(msg_type='test')
+        failed: bool = False
+        try:
+            channel.dispatch(listener_id=my_listener.id, msg=msg)
+            channel.deliver_next(listener_id=my_listener.id)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)